core/src/main/java/org/apache/carbondata/core/indexstore/AbstractMemoryDMStore.java

```diff
@@ -46,7 +48,17 @@ public void finishWriting() {
     // do nothing in default implementation
   }
 
+  public void serializeMemoryBlock() {
+  }
+
+  public void copyToMemoryBlock() {
```

Comment:
why empty implementation?

Reply:
since only for unsafe implementation, its required.